Remotion/Data/DomainObjects.UnitTests/Mapping/PropertyDefinitionObjectMother.cs

```diff
@@ -134,7 +135,7 @@ public static PropertyDefinition CreateForRealPropertyInfo (ClassDefinition clas
           classDefinition,
           fullPropertyName,
           isObjectID,
-          true,
+          propertyInfo.PropertyType.IsNullableType(),
```

Comment:
```suggestion
          NullableTypeUtility.IsNullableType(propertyInfo.PropertyType),
```

Probably need to add the `Remoton.Utilities` using